Automattic/simplenote-ios
# Add Dangermattic configuration

### Fix
This PR adds Dangermattic to this repo. My main goal was to add RuboCop + fix existing violations (see commits breakdown).

I've also used the **same Dangermattic checks as in WordPress iOS**, as per the old [Peril configuration](https://github.com/Automattic/peril-settings/blob/master/peril-settings.json).


### Test
The main validation is to make sure CI is 🟢.

To run Danger yourself:
1. Checkout this branch 
2. Run `bundle install`
3. Run:
```ruby
DANGER_GITHUB_API_TOKEN=<github_token> bundle exec danger pr <PR_URL>
```
You should get in the console the same errors / warnings reported in the PR.


### Release
These changes do not require release notes.
